app/controllers/api/v1/book_suggestion_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module V1
+    class BookSuggestionController < ApplicationController
+      skip_before_action :verify_authenticity_token
+
+      def create
+        bs = BookSuggestion.new(create_params)
+        bs.user_id = current_user ? current_user.id : nil
```

Comment:
```suggestion
        bs.user = current_user ? current_user : nil
```

Prettier to not use the id ;)

Reply:
Oh, I didn't you could do that. Nice.